src/components/pages/Dashboard.tsx

```diff
@@ -42,19 +52,96 @@ export const Dashboard: React.FC = () => {
 
     const [{ originalDeposit }] = useSfStablecoinReducer(reduce, init);
 
-    const { isConnected } = useAccount();
+    const { isConnected, address } = useAccount();
+    const [isTokenAdded, setIsTokenAdded] = useState<boolean | null>(null);
+
+    const { addToken } = useAddToken({
+        debtToken: addresses.debtToken,
+    });
+
+    const checkIfTokenAdded = useCallback(() => {
+        if (!addresses.debtToken) return false;
+        return Boolean(localStorage.getItem(`token_${addresses.debtToken}`));
+    }, [addresses.debtToken]);
+
+    const dismissTokenBanner = () => {
+        if (addresses.debtToken) {
+            localStorage.setItem(`token_${addresses.debtToken}`, 'true');
+        }
+        setIsTokenAdded(true);
+    };
+
+    const handleAddToken = async () => {
+        const success = await addToken();
+        if (success) dismissTokenBanner();
+    };
+
+    useEffect(() => {
+        if (!isConnected || !address || !addresses.debtToken) return;
+
+        const tokenAdded = checkIfTokenAdded();
+        setIsTokenAdded(tokenAdded);
+    }, [isConnected, address, addresses.debtToken, checkIfTokenAdded]);
     return (
-        <div className='mt-[25px] flex w-full flex-col gap-16 px-4 py-8'>
-            <FeatureCardsOrPositions
-                data={{
-                    isConnected,
-                    debtTokenInStabilityPool: data.debtTokenInStabilityPool,
-                    trove: data.trove,
-                    price: data.price,
-                    originalDeposit,
-                }}
-            />
-            <ProtocolOverview data={data} contextData={contextData} />
+        <div className='flex w-full flex-col gap-2 px-4 pt-5 laptop:pt-3'>
+            <div className='w-full'>
+                {!isConnected ? (
+                    <div className='flex flex-col gap-2 p-4'>
```

Comment:
This `div` tag is not needed, so you can remove it and move styles into the parent tag (`<div className='w-full'>`).
L95 as well.